docs/guides/improve-processes-with-optimize.md

```diff
@@ -72,6 +74,20 @@ To create a custom report based on a key performance indicator (KPI) you’d lik
 7. If you’d like, filter the process instance or flow nodes. For example, you can filter by duration, only viewing process instances running for more than seven days.
 8. Finally, you have the option to view particular sets of data from the instance, like instance count or absolute value, by selecting the gear icon to the left of your data customization. You can also choose how you’d like to visualize your data in the box beneath **Visualization** (i.e. bar chart, pie chart, etc.). Once you’ve made your selections, click **Save**.
 
+## Alerts
+
+You don’t have to log in or view reports and dashboards to be alerted that something may need correction or further analysis in your process.
+
+For this purpose, you can create new alerts within your reports. These alerts watch reports for you among collections, and email you an alert if a set outlier occurs in your process flow.
+
+To create an alert, take the following steps:
+
+1. Create a single-value report inside a collection for a KPI you want to track.
```

Comment:
I should start enforcing common terminology. This should actually be a report with a number visualization rather than a single-value report

Reply:
Can add this to the [Components glossary](https://docs.google.com/document/d/1g1mLWK27HCiO_Jf-BT0lw65OBZHx-DzPk3xC8Bh-7Yw/edit) 👍